gloo-edge/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+GLOO_NS=gloo-system
+TRAEFIK_NS=kube-system
+
+# Uninstall default Trafeik resources as gloo-edge and traefik 
+# are mutally exclusive
+
+kubectl config set-context --current --namespace=$TRAEFIK_NS
+helm uninstall traefik
+
+# Get back to default NS
+kubectl config set-context --current --namespace=default
+
+# Add the Helm repository for Gloo Edge
+helm repo add gloo https://storage.googleapis.com/solo-public-helm
+helm repo update
+
+# Create the namespace and install the Helm chart
+kubectl create namespace $GLOO_NS
+helm install gloo gloo/gloo --namespace $GLOO_NS --wait
```

Comment:
I'm still seeing error 
```
Error: Post "http://127.0.0.1:57548/c44c9fe6-da29-435c-9ef7-c03cd9762146/apis/gateway.solo.io/v1/namespaces/gloo-system/gateways": EOF
```

![image](https://user-images.githubusercontent.com/17374859/119599637-0996f700-be03-11eb-87a3-8edfdc2c9adc.png)

Reply:
and this needs to be a seems less install as user will click it once from marketplace and it should install


>
> @DoNnMyTh - I am not able to reproduce the failure step, I had some flaky network issues when i noticed such thing, but since today morning I am not able to see it. To be on safer side I have added `reinstall` step to readme

Reply:
Issue found to be related to Lens Ide app is working fine